README.md

```diff
@@ -13,15 +13,18 @@ requirements.txt                     # Python requirements
 ```
 
 ## Local Environment Setup
+
+To setup a local environment using VMs for testing, [Read here](./local_testing/README.md)
+
 1. Create a virtual environment: `python -m venv venv`
-1. Activate the virtual environment
+2. Activate the virtual environment
    - Windows: `.\venv\Scripts\activate`
     - Note: [Ansible cannot run on Windows hosts natively](https://docs.ansible.com/ansible/latest/user_guide/windows_faq.html#can-ansible-run-on-windows)
    - Linux: `source venv/bin/activate`
-1. Update pip and builder deps: `python -m pip install --upgrade pip wheel setuptools`
-1. Install project dependancies: `python -m pip install -r requirements.txt`
-1. Install the pre-commit hook: `pre-commit install`
-1. Create a `vault_passwords` file and write the vault password to it
+3. Update pip and builder deps: `python -m pip install --upgrade pip wheel setuptools`
+4. Install project dependancies: `python -m pip install -r requirements.txt`
+5. Install the pre-commit hook: `pre-commit install`
+6. Create a `vault_passwords` file and write the vault password to it
```

Comment:
We should leave these as all 1's. This way the markdown renderer does the hard work for us, and if we need to add a number in the middle we don't need to update all the other numbers too.

Reply:
Reverted back to using `1.` for all list items